### DISCUSSION TITLE
Problemik

### DISCUSSION THREAD

**Discussion by Laky73** [ZorTik/AdvancedSQLClient#44] [General]
Dobry deň pan Zortik,
dnes jsem narazil na menší problémik. Jedná se o to, že nejde použít PRIMARY KEY pokud se vytváří tabulka s pouze 1 sloupečkem. 

Ďakujem, cobolko Tlaky.

**Comment by ZorTik** [comment]
@Laky73 Dobrý den,
co přesněji prosím myslíte tím, že nelze použít PRIMARY KEY? Nelze vytvořit tabulkový model, nelze na takovou tabulku zaslat prepared request nebo vyvolat request pomocí mappingu (createProxy)?

**Comment by Laky73** [comment]
![image](https://github.com/ZorTik/AdvancedSQLClient/assets/94743218/2c0af5f4-7583-47b6-8edc-90cd7c244cc2)

Na přiloženém obrázku jsem se pokusil si vytvořit tabulku. Podmínka result3.isSuccessful() ale vyjde na false. Pokud přidám do tabulky další column, tak to funguje.

**Comment by ZorTik** [comment]
Vzhledem k tomu, že Vaše přiložená query je volaná čistě (napřímo) na databázi a neprochází preprocessingem knihovny, bude chyba pravděpodobně někde ve vztahu k databázi.

Pokud vyhodí isSuccessful() false, zkuste prosím logovat chybovou zprávu pomocí
```java
QueryResult result = connection.exec(query);
if (!result.isSuccessful()) {
  String err = result.getRejectMessage();
  // TODO: Log error message
}
```

**Comment by ZorTik** [comment]
@Laky73 Byl problém vyřešen?

**Reply by Laky73** [reply]
Ano, nešlo na column nastavit PRIMARY KEY pokud tam byl TEXT místo VARCHAR.
